README.md

```diff
@@ -21,10 +21,55 @@ other components of OpenShift via support for a baremetal platform type.
 
 # Instructions
 
-## Configuration
+## Preparation
+
+Considering that this is a new install on a clean OS, the next tasks should be performed prior the installation:
+
+1. Enable passwordless sudo for the current user
+
+    `echo "$USER  ALL=(ALL) NOPASSWD: ALL" >/etc/sudoers.d/${USER}`
+
+2. In case of RHEL, invoke `subscription-manager` in order to `register` and `attach` the subscription
+
+3. Install new packages
+
+    `sudo dnf update`
+    `sudo dnf install git make wget tmux jq`
+
+4. Clone the dev-scripts repository
+
+    `git clone https://github.com/openshift-metal3/dev-scripts`
+
+5. Create a config file
 
-Make a copy of the `config_example.sh` to `config_$USER.sh`, and set the
-`PULL_SECRET` variable to the secret obtained from cloud.openshift.com.
+    `cp config_example.sh config_$USER.sh`
+
+6. Configure secrets:
+   1. Download personal pull-secret file from https://cloud.redhat.com/openshift/install/pull-secret
+   2. Get the shared secret from a coworker
+   3. Combine the two files and add them to the user config
+
+    ```bash
```

Comment:
This feels like a script that could live in dev-scripts, eventually.

Reply:
See https://github.com/openshift-metal3/dev-scripts/pull/1093 for a version of dev-scripts that automates more of this.